ibis/sql/alchemy.py

```diff
@@ -28,6 +28,16 @@
 from ibis.client import Database, Query, SQLClient
 from ibis.sql.compiler import Dialect, Select, TableSetFormatter, Union
 
+try:
+    import geoalchemy2 as ga
+    import geoalchemy2.shape as shape
+    import geopandas
+
+    GEO = True
```

Comment:
this is a small thing but maybe something like `is_geo_supported` would be more readable.

Reply:
Sounds good!